spec/requests/api/v1/achievement_request_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe "Api::V1::Achievements", type: :request do
+
+  before(:each) do
+    @achievement1 = Achievement.create!(name: "SimpleCov Pro", criteria: "SimpleCov coverage achieves over 95%")
+    @achievement2 = Achievement.create!(name: "SimpleCov Wizard", criteria: "SimpleCov coverage achieves 100%")
```

Comment:
I love the flair that you gave these achievements and the fact that they are scaled. (and in a more unique way than bronze, silver, and gold!!).